flake.nix

```diff
@@ -0,0 +1,93 @@
+# © 2019 Serokell <hi@serokell.io>
+# © 2019 Lars Jellema <lars.jellema@gmail.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{
+  description = "A Simple multi-profile Nix-flake deploy tool.";
```

Comment:
Looks like copypasta from deploy-rs, could use a better description:  A formatter for Nix code